src/encoded/static/components/dataset.js

```diff
@@ -2209,7 +2226,12 @@ const organismDevelopmentSeriesWormFlyTableColumns = {
             const synch = computeSynch(experiment);
             return <span>{synch}</span>;
         },
-        objSorter: (a, b) => sortMouseAge(a.life_stage_age, b.life_stage_age),
+        objSorter: (a, b) => {
+            if (a.life_stage_age && b.life_stage_age) {
+                return sortMouseAge(a.life_stage_age, b.life_stage_age);
+            }
+            return null;
+        },
```

Comment:
Thank you for suggesting a solution! I tried the fix that you suggested and also tried updating computeSynch() to see if having it return null instead of '' would hide the synchronization column, and I'm not sure why not but it didn't work. Ben said that this is 0 priority though, so I didn't try to track down another solution.